session.go

```diff
@@ -914,14 +909,58 @@ func (s *session) DropPreparedStmt(stmtID uint32) error {
 }
 
 // StmtTxn wraps kv.Transaction to provide a new kv.Transaction.
-// It holds all statement related modification in the buffer before flush to the txn,
+// 1. It holds all statement related modification in the buffer before flush to the txn,
 // so if execute statement meets error, the txn won't be made dirty.
+// 2. It's a lazy transaction, that means it's a txnFuture befort StartTS() is really need.
 type StmtTxn struct {
```

Comment:
After the `future` moved into `StmtTxn`, it's not a statement transaction anymore.
How about renaming it to `TxnState`?